Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.CreateTests.cs

```diff
@@ -98,52 +98,56 @@ public void There_can_only_be_one_active_relationship_between_two_identities()
         acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists");
     }
 
-    [Fact]
-    public void Creating_a_new_relationship_is_possible_if_existing_ones_are_rejected_or_revoked()
+    [Theory]
+    [InlineData(RelationshipStatus.Pending)]
+    [InlineData(RelationshipStatus.Active)]
+    [InlineData(RelationshipStatus.Terminated)]
+    [InlineData(RelationshipStatus.DeletionProposed)]
+    public void There_can_only_be_one_active_relationship_between_two_identities2(RelationshipStatus relationshipStatus)
     {
         // Arrange
         var existingRelationships = new List<Relationship>
         {
-            CreateRejectedRelationship(),
-            CreateRevokedRelationship()
+            CreateRelationshipInStatus(relationshipStatus)
         };
 
         // Act
         var acting = () => new Relationship(RELATIONSHIP_TEMPLATE_OF_1, IDENTITY_2, DEVICE_2, null, existingRelationships);
 
         // Assert
-        acting.Should().NotThrow<DomainException>();
+        acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists");
     }
 
     [Fact]
-    public void Cannot_create_Relationship_if_terminated_Relationship_exists()
+    public void Creating_a_new_relationship_is_possible_if_existing_ones_are_rejected_or_revoked()
     {
         // Arrange
         var existingRelationships = new List<Relationship>
         {
-            CreateTerminatedRelationship()
+            CreateRejectedRelationship(),
+            CreateRevokedRelationship()
         };
 
         // Act
         var acting = () => new Relationship(RELATIONSHIP_TEMPLATE_OF_1, IDENTITY_2, DEVICE_2, null, existingRelationships);
 
         // Assert
-        acting.Should().Throw<DomainException>().WithError("error.platform.validation.relationshipRequest.relationshipToTargetAlreadyExists");
+        acting.Should().NotThrow<DomainException>();
     }
 
     [Fact]
-    public void A_new_relationship_can_be_created_after_decomposing_the_old_one()
+    public void Cannot_create_Relationship_if_terminated_Relationship_exists()
```

Comment:
Wasn't this tested in the first test above?

Reply:
Oh man... I forgot to delete the old tests and rename the new one (it still contains "2" in its name)